checkQC/handlers/qc_handler.py

```diff
@@ -19,33 +22,31 @@ def send(self, value):
 
 
 class QCErrorFatal(object):
-    def __init__(self, msg):
+    def __init__(self, msg, ordering=1):
         self.message = msg
+        self.ordering = ordering
 
     def __str__(self):
-        return "Fatal error: {}".format(self.message)
+        return "Fatal QC error: {}".format(self.message)
 
     def __repr__(self):
         return self.__str__()
 
 
 class QCErrorWarning(object):
-    def __init__(self, msg):
+    def __init__(self, msg, ordering=1):
         self.message = msg
+        self.ordering = ordering
 
     def __str__(self):
-        return "Warning: {}".format(self.message)
+        return "QC warning: {}".format(self.message)
 
     def __repr__(self):
         return self.__str__()
 
 
 class QCHandler(Subscriber):
 
-    handlers = []
```

Comment:
Just like to add that I removed this code since it is not used, and I appear to have forgotten to remove it previously.